ccm_web/client/src/pages/LaunchError.tsx

```diff
@@ -14,6 +14,7 @@ export default function LaunchError (props: LaunchErrorProps): JSX.Element {
   const iOSLink = 'https://www.whatismybrowser.com/guides/how-to-enable-cookies/safari-iphone'
   const firefoxLink = 'https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop?redirectslug=enable-and-disable-cookies-website-preferences'
   const edgeLink = 'https://www.whatismybrowser.com/guides/how-to-enable-cookies/edge'
+  const safariLink = 'https://support.apple.com/guide/safari/prevent-cross-site-tracking-sfri40732/mac'
```

Comment:
nitpick but trying to distinguish 3 safari-related links might be unintuitive. maybe something like `safariMacOSLink`, `safariIOSLink`, `safariCrossSiteTrackingLink`?

Reply:
Done